### PR TITLE
#162948867 Update TravisCI

### PR DESCRIPTION
**Pull request by danuluma**
#### What does this PR do?
Updates TravisCI

#### Description of Task(s) to be completed?
* Create and update travis.yml file
* Update TravisCI to automatically run tests and submit coverage report to codeclimate
* Add Travis badge to ReadMe file

#### How should this be manually tested?
* Navigate to ah-alpha repo on this [branch](https://github.com/andela/ah-alpha-frontend/tree/ch-update-travisci-162948867)
* Click the Travis badge in the ReadMe file

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
[#162948867](https://www.pivotaltracker.com/story/show/162948867)

### CODE REVIEW

**Review by boltonotieno — APPROVED** [review]
LGTM

File: .eslintrc [hunk 1]
```diff
@@ -43,3 +43,4 @@
 
     }
   }
+}
```
Comment (Betty-Kebenei) [inline .eslintrc:44]: Why do we have this added?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,17852 @@
+{
```
Comment (Betty-Kebenei) [inline package-lock.json]: Do you want this back?

Comment (danuluma) [reply package-lock.json]: No, I hadn't confirmed if it was in .gitignore, I have added it now

**Pull request merged**